apps/server/src/modules/account/services/account.service.ts

```diff
@@ -34,6 +39,11 @@ export class AccountService extends AbstractAccountService {
 		}
 	}
 
+	async handle({ deletionRequest }: UserDeletedEvent) {
```

Comment:
please add public/private to each method and the return type in this case Promise<void>

Reply:
done